tickets/models.py

```diff
@@ -0,0 +1,87 @@
+from pyexpat import model
+from django.db import models
+
+class Airline(models.Model):
+    name = models.CharField(max_length=100)
+    logo = models.URLField(max_length=600)
+
+    class Meta:
+        db_table = 'airlines'
+
+class Airplane(models.Model):
+    airline       = models.ForeignKey('Airline', on_delete=models.CASCADE)
+    name          = models.CharField(max_length=100)
+    code          = models.CharField(max_length=100)
+
+    class Meta:
+        db_table = 'airplanes'
+
+class Location(models.Model):
+    name      = models.CharField(max_length=100)
+    code      = models.CharField(max_length=100)
+    image     = models.URLField(max_length=600, null=True)
+    latitude  = models.CharField(max_length=100, null=True)
+    longitude = models.CharField(max_length=100, null=True)
+
+    class Meta:
+        db_table = 'locations'
+
+class FlightInformation(models.Model):
+    airplane                  = models.ForeignKey('Airplane', on_delete=models.CASCADE)
+    departure_location        = models.ForeignKey('Location', on_delete=models.CASCADE, related_name='departure_location')
+    arrival_location          = models.ForeignKey('Location', on_delete=models.CASCADE, related_name='arrival_location')
```

Comment:
related_name을 적용하셨는데, 실제 해당 related_name을 사용했을 때 나오는 데이터를 고려해서 적정한 이름으로 수정해 주세요.

Reply:
현재 제가 생각하는 related_name을 사용했을 때 나오는 데이터는 "location의 객체"라고 생각해 적절한 이름으로 생각됩니다! 어떤 식으로 생각하는 것이 좋을 지 예시를 부탁드려도 될까요?

Reply:
related_name을 지정하면 해당 이름으로 역참조를 하게 됩니다.
즉 Location 입장에서 FlightInformation 정보를 찾게 됩니다.
현재 지정하신 이름으로 하게 되면 아래와 같이 코드가 구성되게 됩니다. 
보시다시피 의미가 이상해져 버립니다.

```python
location = Loaction.objects.get(id=1)
depature_flight_information = location.departure_location.all()
```
아래와 같이 수정 한 후 related_name을 사용할 때 어떤지를 생각해 보면 좋을 것 같아요
```suggestion
    departure_location        = models.ForeignKey('Location', on_delete=models.CASCADE, related_name='departure_flight_informations')
    arrival_location          = models.ForeignKey('Location', on_delete=models.CASCADE, related_name='arrival_flight_informations')
```

Reply:
급하게 생각하느라 놓쳤던 것 같습니다! 설명 감사합니다! 수정 후에 PR 올리도록 하겠습니다.